Source/WebCore/platform/graphics/gstreamer/GStreamerCommon.cpp

```diff
@@ -116,17 +116,15 @@ WTF_MAKE_TZONE_ALLOCATED_IMPL(WebCoreLogObserver);
 
 static GstClockTime s_webkitGstInitTime;
 
-GstPad* webkitGstGhostPadFromStaticTemplate(GstStaticPadTemplate* staticPadTemplate, const gchar* name, GstPad* target)
+WARN_UNUSED_RETURN GstPad* webkitGstGhostPadFromStaticTemplate(GstStaticPadTemplate* staticPadTemplate, const String& name, GstPad* target)
```

Comment:
Why does this need to be a String? An ASCIILiteral would fit better IMO.

Reply:
I would agree, but there is some place where we need to build that string. I might agree with you and use `ASCIILiteral::fromLiteralUnsafe` but I think this solution was better.

Reply:
There's probably more overhead in using a String for all cases, but this is your turf :)

Reply:
Honored.